openshift/templates/visual-verifier/config/inoculation/dev/custom_home.html

```diff
@@ -0,0 +1 @@
+<p> Hello World!</p>
```

Comment:
PLease replace with default page content (in all other `custom_home.html` as well).